File: problemas/1000/1041.cs
using System;

namespace uri1041
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            double x = double.Parse(entrada[0]);
            double y = double.Parse(entrada[1]);
            string quadrante = x == 0 && y == 0
                ? "Origem"
                : x == 0
                    ? "Eixo Y"
                    : y == 0
                        ? "Eixo X"
                        : x > 0 && y > 0
                            ? "Q1"
                            : x < 0 && y > 0
                                ? "Q2"
                                : x < 0 && y < 0
                                    ? "Q3"
                                    : "Q4";
            Console.WriteLine(quadrante);
        }
    }
}

File: problemas/1000/1041.py
(x, y) = map(float, input().split(' '))
quadrante = ''

if x == y == 0:
    quadrante = 'Origem'
elif x == 0:
    quadrante = 'Eixo Y'
elif y == 0:
    quadrante = 'Eixo X'
elif x > 0 and y > 0:
    quadrante = 'Q1'
elif x < 0 and y > 0:
    quadrante = 'Q2'
elif x < 0 and y < 0:
    quadrante = 'Q3'
else:
    quadrante = 'Q4'

print(quadrante)
